includes/admin/views/omise-page-settings.php

```diff
@@ -133,15 +133,8 @@
 								</thead>
 								<tbody>
 									<?php
-									$available_gateways = array(
-										new Omise_Payment_Alipay,
-										new Omise_Payment_Billpayment_Tesco,
-										new Omise_Payment_Creditcard,
-										new Omise_Payment_Installment,
-										new Omise_Payment_Internetbanking,
-										new Omise_Payment_Truemoney
-									);
-									foreach ( $available_gateways as $gateway ) :
+									foreach ( Omise()->payment_methods() as $gateway ) :
+										$gateway = new $gateway;
 										if ( $gateway->is_country_support( $settings['account_country'] ) ) :
```

Comment:
This name should be `is_country_supported`

Reply:
Noted. Thanks for pointing out 👍  
I'll make a change in the later code refactoring PR (as it is not in this scope of change).

Reply:
Similarly, I think I saw `restricted_countries` which is storing a list of country codes that the method is restricted **to**. Those countries then aren't restricted - quite the opposite, they are allowed. This should probably be renamed `restricted_to_countries`

Reply:
IIRC I think I copied the name from somewhere in our plugin projects.
If adding `_to_` makes more sense then maybe we should apply it to every where we are using this as well. 
But noted. Thanks 👍